
--------------------------------------------------
TXLib предупреждает:

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B96C в функции main() в файле Y:/gthb/Functions/main.cpp на строке 5. Попытка записать данные по адресу 0x004FB74A в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

#1: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0x1B40 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B96C in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (5):

      3: int main() {
      4:     char* s = "Pizda";
=>    5:     *customStrchr(s, 'z') = 'w';
      6:     printf("%s", s);
      7: 

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:34:33, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:34:46

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib соболезнует...

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: signal (11, 0x00): SIGSEGV: Доступ по неверному указателю. Ставьте ассерты! 

Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B96C в функции main() в файле Y:/gthb/Functions/main.cpp на строке 5. Попытка записать данные по адресу 0x004FB74A в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

С помощью функции signal() вы можете сами обработать эту ошибку.

#2: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0x1B40 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B96C in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (5):

      3: int main() {
      4:     char* s = "Pizda";
=>    5:     *customStrchr(s, 'z') = 'w';
      6:     printf("%s", s);
      7: 

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:34:33, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:34:46

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib предупреждает:

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B96C в функции main() в файле Y:/gthb/Functions/main.cpp на строке 5. Попытка записать данные по адресу 0x004FB74A в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

#1: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0xBE8 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B96C in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (5):

      3: int main() {
      4:     char* s = "Pizda";
=>    5:     *customStrchr(s, 'z') = 'w';
      6:     printf("%s", s);
      7: 

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:40:52, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:41:11

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib соболезнует...

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: signal (11, 0x00): SIGSEGV: Доступ по неверному указателю. Ставьте ассерты! 

Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B96C в функции main() в файле Y:/gthb/Functions/main.cpp на строке 5. Попытка записать данные по адресу 0x004FB74A в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

С помощью функции signal() вы можете сами обработать эту ошибку.

#2: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0xBE8 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B96C in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (5):

      3: int main() {
      4:     char* s = "Pizda";
=>    5:     *customStrchr(s, 'z') = 'w';
      6:     printf("%s", s);
      7: 

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:40:52, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:41:11

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib предупреждает:

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B978 в функции main() в файле Y:/gthb/Functions/main.cpp на строке 7. Попытка прочесть данные по адресу NULL в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

#1: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0xE04 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B978 in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (7):

      5: 
      6:     char* s = "ZVZVZVZV";
=>    7:     printf("%c", *customStrchr(s, 'z'));
      8: 
      9:     getchar();

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:43:49, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:43:58

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib соболезнует...

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: signal (11, 0x00): SIGSEGV: Доступ по неверному указателю. Ставьте ассерты! 

Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B978 в функции main() в файле Y:/gthb/Functions/main.cpp на строке 7. Попытка прочесть данные по адресу NULL в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

С помощью функции signal() вы можете сами обработать эту ошибку.

#2: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0xE04 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B978 in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (7):

      5: 
      6:     char* s = "ZVZVZVZV";
=>    7:     printf("%c", *customStrchr(s, 'z'));
      8: 
      9:     getchar();

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:43:49, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:43:58

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib предупреждает:

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B978 в функции main() в файле Y:/gthb/Functions/main.cpp на строке 7. Попытка прочесть данные по адресу NULL в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

#1: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0x2634 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B978 in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (7):

      5: 
      6:     char* s = "ZVZVZVZV";
=>    7:     printf("%c", *customStrchr("Z"));
      8:     assert(*customStrchr('z'));
      9: 

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:43:49, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:45:40

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib соболезнует...

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: signal (11, 0x00): SIGSEGV: Доступ по неверному указателю. Ставьте ассерты! 

Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B978 в функции main() в файле Y:/gthb/Functions/main.cpp на строке 7. Попытка прочесть данные по адресу NULL в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

С помощью функции signal() вы можете сами обработать эту ошибку.

#2: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0x2634 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B978 in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (7):

      5: 
      6:     char* s = "ZVZVZVZV";
=>    7:     printf("%c", *customStrchr("Z"));
      8:     assert(*customStrchr('z'));
      9: 

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:43:49, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:45:40

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib предупреждает:

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B978 в функции main() в файле Y:/gthb/Functions/main.cpp на строке 7. Попытка прочесть данные по адресу NULL в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

#1: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0x45C0 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B978 in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (7):

      5: 
      6:     char* s = "ZVZVZVZV";
=>    7:     printf("%c", *customStrchr("Z"));
      8:     assert(customStrchr('z'));
      9: 

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:43:49, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:46:20

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib соболезнует...

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: signal (11, 0x00): SIGSEGV: Доступ по неверному указателю. Ставьте ассерты! 

Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B978 в функции main() в файле Y:/gthb/Functions/main.cpp на строке 7. Попытка прочесть данные по адресу NULL в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

С помощью функции signal() вы можете сами обработать эту ошибку.

#2: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0x45C0 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B978 in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (7):

      5: 
      6:     char* s = "ZVZVZVZV";
=>    7:     printf("%c", *customStrchr("Z"));
      8:     assert(customStrchr('z'));
      9: 

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:43:49, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:46:20

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib предупреждает:

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B978 в функции main() в файле Y:/gthb/Functions/main.cpp на строке 7. Попытка прочесть данные по адресу NULL в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

#1: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0x44E0 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B978 in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (7):

      5: 
      6:     char* s = "ZVZVZVZV";
=>    7:     printf("%c", *customStrchr('Z'));
      8:     assert(customStrchr('z'));
      9: 

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:43:49, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:47:11

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------

--------------------------------------------------
TXLib соболезнует...

Программа: Y:\gthb\Functions\functions.exe,

ВНЕЗАПНО: signal (11, 0x00): SIGSEGV: Доступ по неверному указателю. Ставьте ассерты! 

Нарушение доступа к памяти.

#1: Исключение EXCEPTION_ACCESS_VIOLATION (0xC0000005) при выполнении кода по адресу 0x0042B978 в функции main() в файле Y:/gthb/Functions/main.cpp на строке 7. Попытка прочесть данные по адресу NULL в модуле Y:\gthb\Functions\functions.exe. Спасибо _txVectoredExceptionHandler(), что сообщил. Люблю его <3

С помощью функции signal() вы можете сами обработать эту ошибку.

#2: TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400], Instance: 0x0057569C (32-bit), Flags: Cmdre0, Thread: 0x44E0 (Main), GetLastError(): 126 (Не найден указанный модуль). 

--------------------------------------------------
Стек вызовов:

# 0 0x0042B978 in Y:\gthb\Functions\functions.exe: main() at Y:/gthb/Functions/main.cpp (7):

      5: 
      6:     char* s = "ZVZVZVZV";
=>    7:     printf("%c", *customStrchr('Z'));
      8:     assert(customStrchr('z'));
      9: 

--------------------------------------------------
Developed with:

The Dumb Artist Library (TX Library)
TXLib [Ver: 00173a, Rev: 171, Date: 2022-07-23 02:08:28 +0400]
Copyright (C) Ded (Ilya Dedinsky, http://txlib.ru) <mail@txlib.ru>
See license on: http://txlib.ru

TXLib file:	y:\codeblocks\mingw\include\txlib.h
Compiled:	Aug 26 2022 17:43:49, GNU g++ 4.8.1, std=201103L, MinGW Runtime 3.20, 32-bit, DEBUG
Started:	Aug 26 2022 17:47:11

Run file:	Y:\gthb\Functions\functions.exe
Directory:	Y:\gthb\Functions

--------------------------------------------------
